source/includes/usage-examples/connect-sample-app.php

```diff
@@ -6,11 +6,9 @@
 
 // End example code here
 
-$admin = $client->admin;
-$result = $admin->command(['ping' => 1]);
-
-if ($result) {
+try {
```

Comment:
Should this file demonstrate construction of the `$client` object, or is it intended to be copied _below_ a connection example?

[includes/usage-examples/sample-app-intro.rst](https://github.com/mongodb/docs-php-library/blob/php-standardization/source/includes/usage-examples/sample-app-intro.rst) is included above this, but the bullets there suggest that _this_ content would appear before any of the other connection examples (which instantiate `$client`). That seems backwards.

Reply:
The code examples are supposed to be copied into the highlighted section (lines 5-7), which comes before the try/catch